src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -350,6 +355,9 @@ public void validateMedicalInventoryRow(MedicalInventory inventory, List<Medical
 		if (!errors.isEmpty()) {
 			throw new OHDataValidationException(errors);
 		}
+		String status = InventoryStatus.validated.toString();
```

Comment:
Please change line 250 to:

`LocalDateTime movFrom = inventory.getInventoryDate();`